.travis.yml

```diff
@@ -128,7 +128,7 @@ deploy:
   skip_cleanup: true
   script:
   # only deploy docs in a build after a PR or merge is accepted
-  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_REPO_SLUG" = "OpenMDAO/OpenMDAO" ]; then
```

Comment:
Adding this to make sure that those of us who do verification via Travis on our own repos @ master,( e.g. kmarsteller/OpenMDAO) don't inadvertently push up our docs versions at pull request time.  Approve.